projects/phone-field/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@skyux/phone-field",
+  "version": "5.0.0-beta.1",
+  "peerDependencies": {
+    "@angular/animations": "^12.2.8",
+    "@angular/common": "^12.2.8",
+    "@angular/core": "^12.2.8",
+    "@angular/forms": "^12.2.8",
+    "@skyux/forms": "^5.0.0",
+    "@skyux/i18n": "^5.0.0",
+    "@skyux/indicators": "^5.0.0",
+    "@skyux/lookup": "^5.0.0-beta.2",
+    "@skyux/theme": "^5.0.0"
+  },
+  "dependencies": {
+    "@types/google-libphonenumber": "7.4.21",
```

Comment:
Probably not your fault, but I don't think we need to include `@types/*` in the dependencies section.